kubernetes-addons/aws-for-fluentbit/locals.tf

```diff
@@ -15,8 +16,6 @@ locals {
     namespace                                 = "kube-system"
```

Comment:
We need to move Addons to its dedicated namespaces. Majority of them are suing `kube-system`. We can pick this up a seprate PR

Reply:
agree.